pagopa/pn-frontend
# fix(PN-6118): handled PN_DELIVERY_MANDATENOTFOUND error code

## Short description
Added copy for PN_DELIVERY_MANDATENOTFOUND error code

## List of changes proposed in this pull request
- Copy for PN_DELIVERY_MANDATENOTFOUND

## How to test
On PF user A create a new delegation to user B (on another browser). Then A revoke the delegation and B should see the new error message
